catalog/pom.xml

```diff
@@ -571,6 +571,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-azure-servicebus</artifactId>
```

Comment:
Please can we do these catalog additions as a separate commit. It makes life easier when cherry-picking stuff over to other branches.

Reply:
Done !